### PR TITLE
Ldkfix

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/b282k9qmu2yt7bgg91auhmap2r?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8483)

File: screen/lnd/ldkInfo.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import Button, { ButtonStyle } from '../../components/Button';
 import LdkOpenChannel from './ldkOpenChannel';
 import { Psbt } from 'bitcoinjs-lib';
 import { AbstractWallet, LightningLdkWallet } from '../../class';
+import { sleep } from '../../tests/e2e/helperz';
```
Comment (Overtorment) [inline screen/lnd/ldkInfo.tsx:16]: imported sleep from test helpers kek

**Review by Overtorment — APPROVED** [review]

File: screen/lnd/ldkInfo.tsx (hunk 1 above)
Comment (marcosrdz) [reply screen/lnd/ldkInfo.tsx]: this is what I get for trusting my IDE. changed to context.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/yjtau50c5vth5y7ymajjh2ye7m?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8484)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**